selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/GolangFileEditingTest.java

```diff
@@ -53,6 +61,25 @@
           + "func print() {\n"
           + " fmt.Printf(\"Hello, world. Sqrt(2) = %v\\n\", math.Sqrt(2))\n"
           + "}";
+
+  private static final String REFERENCES_NOTJHING_TO_SHOW_TEXT = "Nothing to show";
+
+  private static final String[] REFERENCES_EXPECTED_TEXT = {
+    "/desktop-go-simple/towers.go\n" + "From:19:5 To:19:10",
```

Comment:
Concatenation at the middle of row looks weird.

Reply:
You propose next:
```
"/desktop-go-simple/towers.go\nFrom:19:5 To:19:10"

Reply:
Yes, I did.